src/test/java/com/faforever/server/game/GameServiceTest.java

```diff
@@ -128,7 +128,7 @@ public void setUp() throws Exception {
       .setMap(map)
       .setFeaturedMod(new FeaturedMod())
       .setVictoryCondition(VictoryCondition.DEMORALIZATION)
-      .setStartTime(Timestamp.from(Instant.now().plusSeconds(999)));
+      .setStartTime(Timestamp.from(Instant.now().plusSeconds(999)).toInstant());
```

Comment:
Euhm....
yeah, I'll also take a look what is currently used in the client overall, that changed multiple times. (it was commited that way on develop, which caused me to just add the autocompletion suggestion. ;)